src/get-started/index.md

```diff
@@ -12,6 +12,20 @@ There are guides to getting started:
 - [with prototyping](prototyping/)
 - [working in production](production/)
 
+## Getting started with the GOV.UK brand refresh
+
+In June 2025, we made updates to the Design System to support a wider refresh of the GOV.UK brand. These include new features in GOV.UK Frontend to help service teams implement GOV.UK’s refreshed brand.
+
+When you set up a new service using GOV.UK Frontend, you’ll need to implement the refreshed brand as part of getting started.
+
+The easiest way to do this is using our Nunjucks page template to add a `govukRebrand` [global value](https://mozilla.github.io/nunjucks/api.html#addglobal) to your environment, with a value of `true`.
+
+This global value makes the affected components use new styles for the brand refresh and display updated assets (such as the refreshed logo in the GOV.UK header component and the crown in the GOV.UK footer component).
+
+We also offer several other ways to do this, depending on the code in your service, however they involve more steps.
```

Comment:
I'm tempted to say the _easiest_ way (especially for non coders) to enable all this is to set the variable on the page template. We'd prefer them to use the global, but I'm wondering if it's a bit technical for the DS website? Probably overthinking it.